src/main/python/RLBotFramework/gui/qt_gui/agent_customisation.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+
+# Form implementation generated from reading ui file 'agent_customisation_3.ui'
+#
+# Created by: PyQt5 UI code generator 5.10
+#
+# WARNING! All changes made in this file will be lost!
+
+from PyQt5 import QtCore, QtGui, QtWidgets
+
+class Ui_AgentPresetCustomiser(object):
```

Comment:
is this generated?

if so can we put it in a folder called generated?
Along with all other files that are automatically generated.